src/dndbeyond/content-scripts/character.js

```diff
@@ -1751,8 +1751,21 @@ function injectRollButton(paneClass) {
         if ($(".ct-health-manage-pane .ct-health-manager__deathsaves").length > 0) {
             if (isRollButtonAdded() || isCustomRollIconsAdded())
                 return;
+            
+            // Check for Advantage/Disadvantage Badges, as Lineages: Reborn advantage on Death Saves or similar will supply
+            const skill_badge_adv = $(".ct-health-manage-pane .ct-health-manager__deathsaves .ddbc-advantage-icon").length > 0;
+            const skill_badge_disadv = $(".ct-health-manage-pane .ct-health-manager__deathsaves .ddbc-disadvantage-icon").length > 0;
+            let deathSaveRollType = RollType.NORMAL;
```

Comment:
It should default to undefined, so it doesn't override the user's own selection.